docs/manual-EN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md

```diff
@@ -0,0 +1,14 @@
+Nebula use `DOWNLOAD` command to get `SST` files from `HDFS`.
+
+Please make sure the `Hadoop` have been installed and `HADOOP_HOME` have been set.
+
+```
+echo $HADOOP_HOME
+```
+
+### Download data
+
+```
+DOWNLOAD HDFS "hdfs://${HADOOP_HOST}:${HADOOP_PORT}/${HADOOP_PATH}"
+```
```

Comment:
`DOWNLOAD` statement will download the sst to /download dir. Could it be configured by users? How many threads are configured to download the files by default? Is it supposed to change by the users?

Reply:
currently default download thread is 3 . It configured by `download_thread_num`